windows/src/developer/TIKE/child/Keyman.Developer.UI.UfrmModelEditor.pas

```diff
@@ -130,13 +130,13 @@       TWordlists = class(TObjectList<TWordlist>)
 
     function GetProjectFile: TProjectFile; override;
 
+  public
     function CanChangeTab(FForward: Boolean): Boolean; override;
     procedure ChangeTab(FForward: Boolean); override;
 
     function CanChangeView(FView: TCodeDesignView): Boolean; override;
     procedure ChangeView(FView: TCodeDesignView); override;
 
-  public
```

Comment:
Interesting change... this intentional for this PR?

Reply:
Yeah, just fixes a compile hint (those functions should have been public because they are public in the parent class).

Just a rider which I dropped in to this PR, no big deal.